pom.xml

```diff
@@ -153,7 +153,12 @@
         <dependency>
             <groupId>org.apache.derby</groupId>
             <artifactId>derbyclient</artifactId>
-            <version>10.15.2.0</version>
+            <version>10.14.2.0</version>
```

Comment:
can we keep newer version of both dependencies here?

Reply:
No. Newer version requires Java9